### ISSUE TITLE
[bug]: `add --path` is broken 

### ISSUE THREAD

**Issue by lacymorrow** [shadcn-ui/ui#6202]
### Describe the bug

I haven't found any combination of arguments to get a custom path out of the add function

https://github.com/shadcn-ui/ui/blob/1081536246b44b6664f4c99bc3f1b3614e632841/packages/cli/src/commands/add.ts#L127

I noticed in the code that it resolves CWD, but passing that as an option as well doesn't work
https://github.com/shadcn-ui/ui/blob/1081536246b44b6664f4c99bc3f1b3614e632841/packages/cli/src/commands/add.ts#L54

### Affected component/components

CLI

### How to reproduce

`npx shadcn@latest add -c . --path src button`

### System Info

```bash
Node 22, NPM 10.9.0
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by DemaPy** [comment]
Hi, did you check this one?

```
Usage: shadcn add [options] [components...]

add a component to your project

Arguments:
  components         the components to add or a url to the component.

Options:
  -y, --yes          skip confirmation prompt. (default: false)
  -o, --overwrite    overwrite existing files. (default: false)
  -c, --cwd <cwd>    the working directory. defaults to the current directory.
  -a, --all          add all available components. (default: false)
  -p, --path <path>  the path to add the component to.
  -h, --help         display help for command
```

**Comment by lacymorrow** [comment]
Yep! That's what I'm referring to, see the reproduction steps. I am unable to get any path to work, the cli defaults to what is in components.jaon

**Comment by ArchiFloyd** [comment]
This happens for me when I try to add a component to a non-existing folder path. I tried adding components via Powershell and on a Arch Linux WSL2 running bash. Both running node lts/iron (20.18.0) and npm (10.8.2). 

If I try to pre-create a folder and point the path argument to that it states that no package.json file was find and then it asks me if I want to start a new Next.js project there. Same happens if i try to point it to a path that end in .tsx.

Either it is broken else the documentation is too scarce

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
